src/Core/src/Platform/iOS/MauiUIApplicationDelegate.cs

```diff
@@ -166,5 +170,18 @@ public virtual void PerformFetch(UIApplication application, Action<UIBackgroundF
 		public IServiceProvider Services { get; protected set; } = null!;
 
 		public IApplication Application { get; protected set; } = null!;
+
+		void ViewDidMoveToWindow(IntPtr @this)
```

Comment:
Every Objective-C method takes the selector as the second parameter. You might not need it in your own implementation, but when calling the original method you must pass it.

Also it's more performant to use [unmanaged function pointers](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/function-pointers), which also makes it possible to use the `[UnmanagedCallersOnly]` attribute.

IMHO the main benefit however is that it makes it much harder to write broken code.

For instance: there are at least two memory problems here:

* You create the `Swizzle` instance in an instance constructor, but store it in a static field. This means that the next time a `MauiUIApplicationDelegate` instance is created, it'll overwrite the existing Swizzle instance.
* The `Swizzle` instance will keep the `MauiUIApplicationDelegate` around forever, because it has a reference to the _instance_ method `ViewDidMoveToWindow`.